google-cloud-tools-plugin/src/com/google/gct/idea/appengine/cloud/DoAppEngineDeployment.java

```diff
@@ -153,7 +153,8 @@ public void undeploy(@NotNull UndeploymentTaskCallback callback) {
         } else {
           logger.error("Deployment process exited with an error. Exit Code:" + event.getExitCode());
           callback.errorOccurred(
-              GctBundle.message("appengine.deployment.error.with.code", event.getExitCode()));
+              GctBundle.message("appengine.deployment.error.with.code", event.getExitCode())
+                  + System.lineSeparator() + GctBundle.message("appengine.deployment.error.old.sdk"));
```

Comment:
Yea I was reading that too. That message just gets displayed within an IJ window - it never gets used in a system dependent way (like doing a system call or something) - so I also \n should be fine.